### PR TITLE
Upgrading package versions to get macOS support

### PR DESCRIPTION
**Pull request by dr-orlovsky**
`npm install` was failing on macOS b/c of outdated package dependencies. This PR fixes the problem.

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     "instascan": "github:shesek/instascan#packaged-lib",
     "jquery": "^3.3.1",
     "path-to-regexp": "^2.4.0",
-    "qruri": "github:shesek/qruri",
+    "qruri": "github:pandoracore/qruri",
```
[the remark this answers is not in the document]
Comment (shesek) [reply client/package.json]: @greenaddress I'm the maintainer of the [`qruri`](https://www.npmjs.com/package/qruri) package. I'm not sure why its installing from github and not from the npm registry, but they should be equivalent.

[the remark this answers is not in the document]
Comment (dr-orlovsky) [reply client/package.json]: I have updated the link in a recent push a week ago. Originally it was created before you have accepted my PR, but now it is fixed

**Comment by dr-orlovsky** [comment]
May I ask you why the PR was declined?

**Comment by shesek** [comment]
I was under the impression that it was only the `qruri`/`canvas` packages that caused issues on macOS, not the other ones. Are you still experiencing issues on macOS after my update to the `qruri` package?

Updating the other dependencies requires some manual review and auditing first, I'll get them updated separately.

**Comment by dr-orlovsky** [comment]
It was not only quri/canvas that was preventing the building; other packages which versions I have updated were also preventing the system from building on macOS

**Comment by shesek** [comment]
The older versions of express, browserify, cssjanus, patch-package, pofile and pug all had bugs that prevented them from working on macOS? This seems unlikely..

Also, why did you add `libffi` as a dependency?

**Comment by dr-orlovsky** [comment]
Ok, with the new macOS build process is broken again even  with all those changes, so I need to figure out how to fix it. The  actual problem happens with C compilation of fsevents library, used by  browserify and babel, and AFAIR the dependencies I made was a solution  to fix it. So let's keep this PR closed and when I will solve npm install bugs for new mac OS I will just open a new PR

**Pull request closed without merging**